CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -11,6 +11,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 - Fix [corruption check may get a `ErrCompacted` error when server has just been compacted](https://github.com/etcd-io/etcd/pull/16048)
 - Improve [Lease put performance for the case that auth is disabled or the user is admin](https://github.com/etcd-io/etcd/pull/16019)
 - Improve [Skip getting authInfo from incoming context when auth is disabled](https://github.com/etcd-io/etcd/pull/16241)
+- Fix [Duplicate gateway url issue](https://github.com/etcd-io/etcd/pull/16490)
```

Comment:
minor comment: it looks a little clearer.
```suggestion
- Fix [Hash and HashKV have duplicated RESTful API](https://github.com/etcd-io/etcd/pull/16490)
```

Reply:
Nice. Updated